File: think-bsdiff/src/main/java/com/think/bsdiff/MainActivity.java
package com.think.bsdiff;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Build;
import android.os.Bundle;
import android.view.View;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        TextView textView  = findViewById(R.id.textView);
        String ver = "版本号为：" + BuildConfig.VERSION_NAME;
        textView.setText(ver);
    }

    public void update(View view) {
        System.out.println("update");
        BsPatcher.test(this);
    }
}

File: think-ui/src/main/java/com/think/ui/MainActivity.kt
package com.think.ui

import android.app.Activity
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button

class MainActivity : Activity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        findViewById<Button>(R.id.button2).setOnClickListener{
            ToastWindow.show(this, "我是toast")
        }
    }
}